lib/rack/jsonp.rb

```diff
@@ -35,8 +36,9 @@ def call(env)
 
       status, headers, response = @app.call(env)
 
-      if callback && headers['Content-Type'] =~ /json/i
+      if callback && callback =~ @callback_regex && headers['Content-Type'] =~ /json/i
         response = pad(callback, response)
+        headers['Content-Type-Options'] = 'nosniff'
```

Comment:
I missed this in my first review. I think this is supposed to be `X-Content-Type-Options`?

https://www.owasp.org/index.php/OWASP_Secure_Headers_Project#xcto